charts/consul/templates/api-gateway-controller-deployment.yaml

```diff
@@ -56,7 +59,7 @@ spec:
       serviceAccountName: {{ template "consul.fullname" . }}-api-gateway-controller
       containers:
       - name: api-gateway-controller
-        image: {{ .Values.apiGateway.image }}
+        image: {{ default .Values.global.imageK8S .Values.apiGateway.image}}
```

Comment:
This *should* default the image to be the globalK8S image, and only insert the API Gateway image if it is defined.